src/bmad_loop/adapters/opencode_http.py

```
@@ -1,42 +1,786 @@
+
+    def send_text(self, handle: SessionHandle, text: str) -> None:
+        """Nudge the running session. Best-effort: a server that died between
+        the liveness probe and the nudge is caught as `crashed` on the next
+        tick, not by blowing up the completion loop."""
+        sess = self._sessions.get(handle.task_id)
+        if sess is None:
+            return
+        try:
+            self._prompt(sess, text)
+        except Exception:  # noqa: BLE001  # nosec B110 - next tick's poll() settles liveness
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)